src/main/groovy/life/qbic/datamodel/datasets/datastructure/folders/nanopore/Pod5SkipFolder.groovy

```diff
@@ -0,0 +1,38 @@
+package life.qbic.datamodel.datasets.datastructure.folders.nanopore
+
+import life.qbic.datamodel.datasets.datastructure.files.nanopore.Pod5File
+import life.qbic.datamodel.datasets.datastructure.folders.DataFolder
+
+/**
+ * A special case of a DataFolder, its name is always pod5_skip.
+ *
+ * Its children field contains either a list of type List<Pod5Files>
```

Comment:
the either makes no sense - can it contain (barcoded?) folders?

Reply:
Nice Catch it's a copy paste error from my side 😓

Reply:
AFAIK the skip folders only contain pod5 files 
![Bildschirmfoto 2023-11-03 um 15 22 55](https://github.com/qbicsoftware/data-model-lib/assets/29627977/9526e5ef-52fc-4f48-80da-998630b56fed)